CellFabric/tests/test_canvas_drc.py

```diff
@@ -0,0 +1,73 @@
+from cell_fabric import Canvas, Pdk, Wire
+
+def test_min_length_pass():
```

Comment:
You can use a fixture to save repeat the first three lines each of these tests. See `tests/test_ascii_sticks.py`